HISTORY.rst

```diff
@@ -64,6 +64,9 @@ Unreleased Changes
       flag, leading to a perceived buffering of spatial criteria in certain
       cases.  In InVEST 3.9.0, these were rasterized with ``ALL_TOUCHED=FALSE``.
       https://github.com/natcap/invest/issues/1120
+* Stormwater
```

Comment:
Technically we list this model as "Urban Stormwater Retention" in the key at the top of this file.